msvc-full-features/Cataclysm-common.props

```diff
@@ -67,6 +88,56 @@
       <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
       <StripPrivateSymbols>$(OutDir)$(TargetName).stripped.pdb</StripPrivateSymbols>
     </Link>
+    <Link Condition="$(_CDDA_USE_LLD_LINK)">
+      <AdditionalLibraryDirectories Condition="$(Configuration.StartsWith(Debug))">
+        $(ProjectDir)vcpkg_installed\$(VcpkgTripet)\$(VcpkgTriplet)\debug\lib;%(AdditionalLibraryDirectories)
+      </AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories Condition="$(Configuration.StartsWith(Release))">
+        $(ProjectDir)vcpkg_installed\$(VcpkgTripet)\$(VcpkgTriplet)\lib;%(AdditionalLibraryDirectories)
+      </AdditionalLibraryDirectories>
+      <AdditionalDependencies>
+        brotlicommon-static.lib;
+        brotlidec-static.lib;
+        brotlienc-static.lib;
+        bz2$(VcpkgLibSuffix).lib;
+        FLAC.lib;
```

Comment:
We should probably hide SDL sound related libs behind `SDL_SOUND`

Reply:
If the symbols aren't needed because they're not compiled into the binary, then listing these extra libs should have no effect on the binary.